.cirrus.yml

```diff
@@ -46,12 +52,12 @@ linux_1_cpu_1G_template: &LINUX_1_CPU_1G
     cpu: 1
     memory: 1G
 
-linux_2_cpu_4G_java_17_template: &LINUX_2_CPU_4G_JAVA_17
+linux_2_cpu_6G_java_17_template: &LINUX_2_CPU_6G_JAVA_17
```

Comment:
This is related to the java OOM errors, or why do we need more power?

Reply:
Yes It's relate OOM error